.git-blame-ignore-revs

```diff
@@ -1,3 +1,6 @@
 # first run of clang-format over MueLu
 1b1b032cf67a001d7c7595e3ccea653d5bcd1277
-27387a814f72487f5a3fbb634cd5d1b298dfe4c5
\ No newline at end of file
+27387a814f72487f5a3fbb634cd5d1b298dfe4c5
+# clang-format for Tempus
+586c87a8f1d21a837401aed54973689ac1177fba
+a2a4c5895dfc0f57f54b1238fd97d64d74c897a2
```

Comment:
You don't need the `a21` one. That commit only changed the workflow file. The first one is the one to ignore since it has a gazillion lines of changes.

Reply:
Gotcha.  Can you re-review?  :)